### DISCUSSION TITLE
Leaflet draw controls on Nuxt 3

### DISCUSSION THREAD

**Discussion by niddu85** [nuxt/nuxt#17502] [Questions]
I am trying to add draw/edit controls to leaflet maps on Nuxt 3

I was able to add vue-leaflet following the suggestions: https://github.com/nuxt/framework/discussions/2363

but there is no way to enable editing 

can somebody please help me with that?

**Comment by wobsoriano** [comment]
Check this one https://github.com/vue-leaflet/vue-leaflet/issues/145#issuecomment-1046197874
